integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomainModelInPV.java

```diff
@@ -340,14 +341,21 @@ private static V1Pod setupWebLogicPod(String namespace) {
     createSecretForBaseImages(namespace);
 
     final String podName = "weblogic-pv-pod-" + namespace;
+    String argCommand = "chown -R 1000:0 " + modelMountPath;
```

Comment:
This code block is repeated several times. Can it be consolidated to a function somewhere?

Reply:
consolidated as util method